### PR TITLE
Jelly 80 dev

### PR DESCRIPTION
**Pull request by be-nny**
- improved test coverage to 91.4%
- removed redundant and unused toolkit functions
- updated readme docs

### CODE REVIEW

File: FunctionContext.go [hunk 1]
```diff
@@ -30,30 +28,20 @@ type FunctionContext struct {
...
 
-// Creates a json object in the json format. Example:
-/*
-tk.Json{
-"foo":"bar"
-}
-*/
-// Results in an object which serializes to
-/* { "foo":"bar" }*/
-type Json map[string]any
```
Comment (XoToM) [inline FunctionContext.go:54]: Why was this removed?

Comment (be-nny) [reply FunctionContext.go:54]: its not necessary to have it. it obscures traditional coding practices and its name is confusing with the package json name
